apps/dot_ext/views/authorization.py

```diff
@@ -199,7 +200,7 @@ def __init__(self, version=1):
     def dispatch(self, request, uuid, *args, **kwargs):
         # Get auth_uuid to set again after super() return. It gets cleared out otherwise.
         auth_flow_dict = get_session_auth_flow_trace(request)
-        # trows DoesNotExist
+        # throws DoesNotExist
```

Comment:
Comment is unnecessary, I can see that it throws `DoesNotExist` from the except :)

Reply:
some residue code lines